src/config.ts

```diff
@@ -60,6 +60,10 @@ export const MUTECT_REPO_URL =
   process.env.MUTECT_REPO_URL ||
   "https://github.com/icgc-argo/gatk-mutect2-variant-calling.git";
 
+export const OPEN_ACCESS_REPO_URL =
+  process.env.OPEN_ACCESS_REPO_URL ||
+  "https://github.com/icgc-argo-workflows/open-access-variant-filtering";
```

Comment:
This repo URL currently only works with the `.git` omitted, making it different from the rest of the repo URLs. There is a ticket to fix the cause of this issue here: https://github.com/icgc-argo/workflow-roadmap/issues/211